### PR TITLE
Remove deprecated nodes

### PR DESCRIPTION
**Pull request by catlee**

### CODE REVIEW

File: manifests/moco-nodes.pp [hunk 1]
```diff
@@ -229,391 +188,6 @@
...
 
-# Signing workers
-node /^signingworker-.*\.srv\.releng\..*\.mozilla\.com$/ {
-    $aspects       = [ 'maximum-security' ]
-    $only_user_ssh = true
-    include toplevel::server::signingworker
-}
-
-# Signing scriptworkers
-node /^signing-linux-\d*\.srv\.releng\..*\.mozilla\.com$/ {
```
Comment (escapewindow) [inline manifests/moco-nodes.pp]: We still use this for xpi, but it's pinned, so maybe we'll just keep that puppet env forked off an older rev.

File: manifests/moco-nodes.pp [hunk 2]
```diff
@@ -229,391 +188,6 @@
...
-    $aspects                  = [ 'maximum-security' ]
-    $signing_scriptworker_env = 'prod'
-    $timezone                 = 'UTC'
-    $only_user_ssh            = true
-    include toplevel::server::signingscriptworker
-}
-
-node /^depsigning-worker.*\.srv\.releng\..*\.mozilla\.com$/ {
```
Comment (escapewindow) [inline manifests/moco-nodes.pp]: Same here.

File: manifests/moco-nodes.pp [hunk 3]
```diff
@@ -229,391 +188,6 @@
...
-node /^tb-tree-comm-\d+\.srv\.releng\..*\.mozilla\.com$/ {
-    $aspects          = [ 'maximum-security' ]
-    $treescriptworker_env = 'tb-comm-prod'
-    $timezone         = 'UTC'
-    $only_user_ssh    = true
-    include toplevel::server::treescriptworker
-}
-
-## Loaners
-
```
Comment (escapewindow) [inline manifests/moco-nodes.pp]: maybe check with the related relops folk?

File: manifests/moco-nodes.pp (hunk 1 above)
Comment (catlee) [reply manifests/moco-nodes.pp]: yeah, or just deploy directly to the worker. it will be moving to GCP soon, right?

Comment (escapewindow) [reply manifests/moco-nodes.pp]: +1

**Comment by dividehex** [comment]
👍 That all looks good to me

**Pull request merged**